Nos/Service/Relay/RelaySubscriptionManager.swift

```diff
@@ -166,11 +166,40 @@ actor RelaySubscriptionManagerActor: RelaySubscriptionManager {
     // MARK: - Talking to Relays
     
     func processSubscriptionQueue() async {
-        let waitingLongSubscriptions = all.filter { !$0.isOneTime && !$0.isActive }
-        let waitingOneTimeSubscriptions = all.filter { $0.isOneTime && !$0.isActive }
-        
-        waitingOneTimeSubscriptions.forEach { start(subscription: $0) }
-        waitingLongSubscriptions.forEach { start(subscription: $0) }
+        var waitingLongSubscriptions = [RelaySubscription]()
+        var waitingOneTimeSubscriptions = [RelaySubscription]()
+        var activeSubscriptionsCount = [URL: Int]()
+
+        all.forEach { relaySubscription in
```

Comment:
This is a little refactor to avoid iterating over `all` more than one time, the code is not nicer, but it should be a bit more performant